### DISCUSSION TITLE
Welcome to ONNX Runtime discussions!

### DISCUSSION THREAD

**Discussion by natke** [microsoft/onnxruntime#4847] [General]
Please add your question or comment here.

If you are reporting a bug please [create an issue](https://github.com/microsoft/onnxruntime/issues/new/choose)

**Comment by Rowlando13** [comment]
Hi! I have been working with ORT's quantization functionality. It has been super useful. I would like to try it out on models larger than 2gb but I have seen the warning 
"As of onnxruntime 1.4.0, models larger than 2GB will fail to quantize due to protobuf constraint.
This limitation will be removed in the next release of onnxruntime." 
I was wondering if this was still planned and if there was a timeline for the next release?
